core/src/main/scala/org/scalawag/jibe/backend/ubuntu/IsPackageInstalled.sh

```diff
@@ -0,0 +1,29 @@
+PATH=/bin:/usr/sbin:/usr/bin
+
+# Find out if the package is already installed, what version it is, and what the next available version is:
+CURRENT_PACKAGE_VERSION=$(dpkg-query -W --showformat='${Version}\n' $package_name 2> /dev/null)
+CANDIDATE_PACKAGE_VERSION=$(apt-cache policy $package_name | grep Candidate | sed "s/  Candidate: //")
+
+# First validate the input:
+# 1. Complain if the package_name is empty
+# 2. Complain if the a package of that name doesn't exist
+#
+# Next, return 1 if we need to install this package, i.e.:
+# 1. If the package is not installed
+# 2. If a target package version was passed in and the current version doesn't match it
+# 3. If a new version is available that's greater than the current version
+
+if [[ "" == "$package_name" ]]; then
+    echo "No package_name provided"
+    exit 0
+elif [[ "" == $(apt-cache search --names-only "^$package_name$" | grep "^$package_name -") ]]; then
```

Comment:
I'm getting an exit code of 0 whether the packages exists or not from apt-cache showpkg.